### DISCUSSION TITLE
Tidak dapat kirim WA ke Group...

### DISCUSSION THREAD

**Discussion by yefridho** [kirimwa-id/forum#186] [Tanya Jawab]
Mas Rio, sebelumnya kirim WA ke group lancar tidak ada masalah.... tapi sekarang kirim pesan ke group lagi ada masalah... status pesan pending... tapi pesannya tidak masuk ke Group... ini baru terjadi 2 hari ini aja.... sebelumnya lancar... Mohon di cek mas Rio yang pesan ke Groupnya ya.... Makasih Mas...

**Comment by yefridho** [comment]
Up... mohon dibantu ya mas...

**Reply by rioastamal** [reply]
Coba langkah berikut:

- Logout dari linked device
- Connect ulang ke API
- Kirim pesan ke group lewat API, jika pesan masih pending, ulangi pengiriman pesan ke group 1-2x harusnya bisa menjadi solusi masalah tersebut.

**Comment by yefridho** [comment]
Mas Rio.

Untuk WA ke group terjadi lagi setelah 1.5 tahun, akhirnya terjadi lagi. tidak ada perubahan kodingan sama sekali.
dan juga sudah remove link device 2 kali dan scan ulang QR Code 2 kali, tapi masalah ini terjadi lagi sekarang.
mohon bantuannya.

**Comment by yefridho** [comment]
Mas Rio

MOHON BANTUAN PLEASE, URGENT MAS...

Untuk WA ke group terjadi lagi setelah 1.5 tahun, akhirnya terjadi lagi. tidak ada perubahan kodingan sama sekali.
dan juga sudah remove link device 2 kali dan scan ulang QR Code 2 kali, tapi masalah ini terjadi lagi sekarang.
mohon bantuannya.

**Comment by yefridho** [comment]
status pesan error nya begini mas Rio. sudah remove device di link device, dan scan QR lagi. udah 10 kali scan ulang QR tetap tidak bisa. untuk pesan personal bisa tapi group ngak bisa. udah bikin device baru, tapi untuk ke group tetap tidak bisa, hanya bisa personal.

{"id":"kwid-6216758e9cda4ddd8c6b2d2df3e",
"message":"Can't send message. Error: SessionError: No sessions",
"status":"fail",
"created_at":"2025-07-30T23:57:56.831Z",
"payload":{"message":"xxx",
	"phone_number":"xxx",
	"device_id":"xxx",
	"message_type":"text",
	"caption":null,
	"is_group_message":true,
	"send_at":"now"}
}

**Comment by rioastamal** [comment]
Pindah ke https://github.com/kirimwa-id/forum/discussions/207#discussion-8657827
